share/site/duckduckgo/styleguide.tx

```diff
@@ -21,7 +21,6 @@
 				<li><a href="#txt-n-color"><: l("Text & Colors") :></a></li>
 				<li><a href="#forms"><: l("Forms") :></a></li>
 				<li><a href="#grids"><: l("Grids") :></a></li>
-				<li><a href="#popovers">Popovers</a></li>
```

Comment:
It's probably worth addressing in the future, but for now i was just removing the broken section from this page.  If we can fix this in the future then we can just rescue it from the history. :)